indico/modules/events/sessions/templates/management/_session_list.html

```diff
@@ -0,0 +1,110 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_session_list(event, sessions, default_colors) -%}
+    {% if sessions -%}
+        <table class="i-table tablesorter">
+            <thead>
+                <tr class="i-table">
+                    <th class="i-table checkbox-column" data-sorter="false"></th>
+                    <th class="i-table id-column">{% trans %}ID{% endtrans %}</th>
+                    <th class="i-table">{% trans %}Title{% endtrans %}</th>
+                    <th class="i-table">{% trans %}Code{% endtrans %}</th>
+                    <th class="i-table" data-sorter="false">{%- trans %}Type{% endtrans -%}</th>
+                    <th class="i-table">{% trans %}Colour{% endtrans %}</th>
+                    <th class="i-table" data-sorter="false">{% trans %}Blocks{% endtrans %}</th>
+                    <th data-sorter="false"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for session in sessions -%}
+                    <tr class="i-table">
+                        <td class="i-table">
+                            <input class="select-row" type="checkbox" value="{{ session.id }}">
+                        </td>
+                        <td class="i-table">#{{ session.friendly_id }}</td>
+                        <td class="i-table"><strong>{{ session.title }}</strong></td>
+                        <td class="i-table">{{ session.code }}</td>
+                        <td class="i-table">
+                            {%- if session.is_poster %}
+                                {%- trans %}Poster{% endtrans -%}
+                            {% else %}
+                                {%- trans %}Standard{% endtrans -%}
+                            {% endif -%}
+                        </td>
+                        <td class="i-table">
+                            <div class="group">
+                                <span class="js-color-switch"
```

Comment:
Why not having `js-color-switch` directly on the trigger element? You can make the widget look for `switch-trigger` and if not found use the element.